.cargo/config.toml

```diff
@@ -4,3 +4,11 @@ linker = "aarch64-linux-gnu-gcc"
 [alias]
 dev-maker = "run --bin maker -- testnet"
 dev-taker = "run --bin taker -- --maker localhost:9999 --maker-id 10d4ba2ac3f7a22da4009d813ff1bc3f404dfe2cc93a32bedf1512aa9951c95e testnet" # Maker ID matches seed found in `testnet/maker_seed`
+
+# Inspired by https://github.com/EmbarkStudios/rust-ecosystem/pull/68.
```

Comment:
yup, this seems to be the most common way people configure lints in workspaces...